htdocs/core/menus/standard/eldy.lib.php

```diff
@@ -648,7 +648,7 @@ function print_left_eldy_menu($db,$menu_array_before,$menu_array_after,&$tabMenu
 				if ($usemenuhider || empty($leftmenu) || $leftmenu=="prospects") $newmenu->add("/societe/list.php?type=p&amp;sortfield=s.datec&amp;sortorder=desc&amp;begin=&amp;search_stcomm=2", $langs->trans("LastProspectContactInProcess"), 2, $user->rights->societe->lire);
 				if ($usemenuhider || empty($leftmenu) || $leftmenu=="prospects") $newmenu->add("/societe/list.php?type=p&amp;sortfield=s.datec&amp;sortorder=desc&amp;begin=&amp;search_stcomm=3", $langs->trans("LastProspectContactDone"), 2, $user->rights->societe->lire);
 				*/
-				$newmenu->add("/societe/card.php?leftmenu=prospects&amp;action=create&amp;type=p", $langs->trans("MenuNewProspect"), 2, $user->rights->societe->creer);
+				$newmenu->add("/societe/card.php?leftmenu=prospects&amp;action=create&amp;client=2", $langs->trans("MenuNewProspect"), 2, $user->rights->societe->creer);
```

Comment:
if SOCIETE_DISABLE_PROSPECTSCUSTOMERS is not set (so default setup) if you use client=2, you will have form preselected on customer only when we want a default value to be preselected on prospect+customer

Reply:
With or without this option, when we click on "new prospect" into menu, the field type should be preselected to prospect and client on "new customer " the field should be preselected to customer.
Should I define this default "new customer" preselected type into setup (for Dolibarr 10 then) ?

Reply:
may be this #10514 ?